src/components/SectionNav/SectionNav.tsx

```diff
@@ -84,13 +66,16 @@ const createNavItems = (
 
 export const SectionNav: React.SFC<Props> = ({
     keyConsts,
-    sections,
     style,
     className,
-    onNavigation
+    onNavigation,
+    navItemClassName,
+    navWrapperClassName
 }) => (
-    <nav className={[styles.component, className].join(" ")} style={style}>
-        {createNavItems(sections, keyConsts, onNavigation)}
+    <nav className={cx(className)} style={style}>
```

Comment:
Any reason to wrap a single classname with className?

Reply:
I vacillated and decided that having it there in concordance with the convention was appropriate, and there is a possibility that we may want to add some definition styles after all.

Happy to go either way on this though

Reply:
I think I should convert the name `cx` to `classNames` as elsewhere though